pkg/processor/reportprocessor.go

```diff
@@ -320,7 +320,10 @@ func getSevFromTemplate(plc unstructured.Unstructured, name string) string {
 			return severity
 
 			// If this isn't an OCM policy, check for a severity annotation
-		} else if severityAnnotation, ok := plc.GetAnnotations()["policy.open-cluster-management.io/severity"]; ok {
+		} else {
+			severityAnnotation, _, err := unstructured.NestedString(objDef, "metadata", "annotations", "policy.open-cluster-management.io/severity")
+			glog.V(1).Infof("error parsing objectDefinition severity annotation as a string for policy %s, template %d: %s", plcName, idx, err)
```

Comment:
Shouldn't this be guarded by an `if err != nil {`?

Reply:
It could, but `NestedString()` would set `severityAnnotation` to an empty string, so the end result would be the same since that's the default return value.

Reply:
But the code would probably be easier to read and clearer. I'll adjust it.

Reply:
Haha, oh I see what you mean now! Oops. Well, it's fixed! 🤣